config/dev.js

```diff
@@ -21,11 +20,9 @@ module.exports = {
         'data:',
       ],
       scriptSrc: [
-        "'self'",
         amoCDN,
       ],
       styleSrc: [
-        "'self'",
         amoCDN,
       ],
```

Comment:
Do the dev and stage configs need `mediaSrc: [amoCDN]` so that the URL is updated from the prod URL?